website/content/en/blog/the-economics-of-the-cncf-platform-engineering-model.md

```diff
@@ -0,0 +1,84 @@
+
+## The theory of platforms
+
+In economics platforms are modelled as [two or multi-sided markets](https://en.wikipedia.org/wiki/Two-sided_market). Basic economics speaks about markets as the interaction of suppliers and consumers. A two-sided market might still have suppliers and consumers, but the market itself is a product with its own market. A few well-known two-sided market products are operating systems or social networks. There is a market for operating systems that consumers choose. At the same time there is a market for operating systems that application developers choose. The more applications are available for a given operating system, the more users want to choose it; and the more users use an operating system, the more applications are developed for it. Interestingly, this aspect of two-sided products brings interesting market dynamics with it.
+
+While physical markets might have similar characteristics, because of the nature of the physical world (travel is costly) the two-sided market considerations are often negligible. On the other hand, for digital goods a two-sided market suffers for several properties that need regulatory intervention.
+
+To summarise the economics: In two or multi-sided markets, the products are platforms with users and suppliers who evaluate the platform: the more or higher quality users make a platform more compelling to enter for the suppliers; the more or higher quality suppliers make the platform more compelling to enter for the users.
+
+For the rest of the article, I’ll speak about multi-sided markets as they better capture the world of DevOps platforms, and the economics in our case is very similar to two-sided markets.
+
+## Application to platform engineering
+
+DevOps platforms or Internal Developer Platforms (IDP) operated by platform engineering teams have many characteristics of multi-sided markets. For a software developer there is no reason to join an IDP if it does not improve the developer experience beating the experienced cost of switching. Similarly, for different [platform teams](https://teamtopologies.com/key-concepts) in the company there is no reason to invest into an IDP without having developers using it. This shows one important aspect of multi-sided markets. Adoption looks like a chicken and egg problem. Before discussing adoption in detail, let’s speak a bit about the many sides of a DevOps platform.
+
+I called these platforms many sided. The [CNCF Platform White Paper](https://tag-app-delivery.cncf.io/whitepapers/platforms/#why-platforms) describes the motivations why enterprises might invest into platforms:
+
+“The desire to refocus delivery teams on their core focus and reduce duplication of effort across the organisation has motivated enterprises to implement platforms for cloud-native computing. By investing in platforms, enterprises can:
+
```

Comment:
I'm struggling to understand how this is a two-sided market problem. A platform team can be expressing opinions, like focusing on some golden paths, that are the recommended approach to specific problems. This seems more like a highly curated store than a true two-sided market to me.

Reply:
Two-sided markets are not defined by their offering, but by the market dynamics.

Engineers do their work with or without a platform. Why would they switch to a platform? Because it offers them value. The more value it offers, the more engineers switch. The more engineers switch, the better the platform can become. The more features the platform supports, the harder it is to replace it. 

This is the two-sided market dynamics. As you increase usage on one side of the market, there is more reason to increase usage (in this case support) on the other side of the market).

Reply:
@abangser I like your Netlifx - Etsy examples a lot, and I agree with everything in your comment except for the last half-sentence.

> the fact that support would be increased, it is maybe pointing more towards a one-sided market since it is the producers of the platform that are increasing investment, not a different platform consumer/producer doing so.

This will really depend on the organisation structure. Who owns the platform (e.g. Backstage installation) vs Who owns the scaffolding templates and integrations? If these are owned by the same team, I agree with you. If these are owned by different teams, then the more users use the platform as consumers, the more compelling it is for "supplier" teams to join the platform. Support of the platform itself does not increase, the platform owners job does not change.